src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -51,6 +53,8 @@ function ActionCell({
     const text = translate(actionTranslationsMap[action]);
 
     const shouldUseViewAction = action === CONST.SEARCH.ACTION_TYPES.VIEW || (parentAction === CONST.SEARCH.ACTION_TYPES.PAID && action === CONST.SEARCH.ACTION_TYPES.PAID);
+    const isApproveOrPaidAction = action === CONST.SEARCH.ACTION_TYPES.APPROVE || action === CONST.SEARCH.ACTION_TYPES.PAID;
+    const shouldUseSuccessStyle = isApproveOrPaidAction ? !isOnHold : true;
```

Comment:
We should remove this logic and just use the `shouldUseSuccessStyle` passed as a prop. Additionally, the `Paid` or any of the `Badge` actions shouldn't change styles, since those reports are either paid or done already, which means that there are no more actions to be taken. The only part that cares about this is the instance in line 106.